RadixWallet/Features/DappInteractionFeature/Coordinator/DappInteractionFlow.swift

```diff
@@ -515,10 +515,10 @@ extension DappInteractionFlow {
 			_ signedSubintent: SignedSubintent,
 			_ expiration: DappToWalletInteractionSubintentExpiration
 		) -> Effect<Action> {
-			let preAuthResponse = newWalletToDappInteractionPreAuthorizationResponseItems(signedSubintent: signedSubintent)
+			let preAuthResponse = newWalletToDappInteractionPreAuthorizationResponseItems(signedSubintent: signedSubintent, expirationTimestamp: expiration.timestamp)
```

Comment:
signedSubintent contains the expiration timestamp already, no?

Reply:
no, only the request contains the expiration timestamp (which we have now included in the response model as well).

Reply:
the response contains the calculated expiration timestamp, why not use?

Reply:
Updated here and in https://github.com/radixdlt/sargon/pull/282